e2e/cypress/tests/integration/team_settings/closed_team_invite_by_email_spec.js

```diff
@@ -39,6 +39,9 @@ describe('Team Settings', () => {
         cy.apiUpdateConfig({
             LdapSettings: {Enable: false},
             ServiceSettings: {EnableOnboardingFlow: true},
+            TeamSettings: {
+                EnableOpenServer: false,
+            },
```

Comment:
Did this test catch the issue before your fix?

Reply:
Correct.

Reply:
I see, that makes sense. Thanks!